src/containers/Airdrop/AirdropGiftTracker/sections/LoggedIn/segments/Invite/Invite.tsx

```diff
@@ -84,7 +65,7 @@ export default function Invite() {
                 </GemPill>
             </InviteButton>
 
-            {nextBonusTier && (
+            {/*nextBonusTier && (
```

Comment:
i'd personally just remove the commented out code entirely

Reply:
done